cmd/lotus-miner/proving.go

```diff
@@ -291,13 +297,27 @@ var provingDeadlinesCmd = &cli.Command{
 			if di.Index == uint64(dlIdx) {
 				cur += "\t(current)"
 			}
-			_, _ = fmt.Fprintf(tw, "%d\t%d\t%d (%d)\t%d%s\n", dlIdx, partitionCount, sectors, faults, provenPartitions, cur)
+
+			_, _ = fmt.Fprintf(tw, "%d\t%s\t%d\t%d (%d)\t%d%s\n", dlIdx, deadlineOpenTime(head, uint64(dlIdx), di),
+				partitionCount, sectors, faults, provenPartitions, cur)
 		}
 
 		return tw.Flush()
 	},
 }
 
+func deadlineOpenTime(ts *types.TipSet, dlIdx uint64, di *dline.Info) string {
+	// 30 minutes a deadline
+	thirtyMinutes := uint64(30 * 60)
```

Comment:
We should use `di.WPoStProvingPeriod/di.WPoStPeriodDeadlines` to get the number of epochs between deadlines. That way, this'll work on test networks with shorter block times.